code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -96,8 +96,8 @@
 	foldable = null //special form fitted boxes should not be foldable.
 
 /obj/item/storage/box/survival
-	name = "crew survival kit"
-	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and white stripe indicate this one contains oxygen."
+	name = "site survival kit"
+	desc = "A survival box containing a variety of supplies. This one contains oxygen, as shown by the red panel and two striped yellow lines."
```

Comment:
```suggestion
	desc = "A box containing a variety of supplies. Contains basic emergency kit, as indicated by the red panel and two striped yellow lines."
```

Reply:
you didn't even make this grammatically correct.